third_party/terraform/website/docs/r/service_networking_connection.html.markdown

```diff
@@ -49,3 +49,11 @@ The following arguments are supported:
 * `reserved_peering_ranges` - (Required) Named IP address range(s) of PEERING type reserved for
   this service provider. Note that invoking this method with a different range when connection
   is already established will not reallocate already provisioned service producer subnetworks.
+
+## Import 
```

Comment:
nit: can you match the format/wording of one of the generated resources like https://www.terraform.io/docs/providers/google/r/compute_address.html#import?